src/libs/actions/User.js

```diff
@@ -103,11 +112,46 @@ function setSecondaryLogin(login, password) {
     });
 }
 
+/**
+ * Validates a login given an accountID and validation code
+ *
+ * @param {Number} accountID
+ * @param {String} validateCode
+ * @returns {Promise}
```

Comment:
Didn't see anything using this response so assuming we maybe don't need to return anything from here.